src/Interpreters/DatabaseCatalog.cpp

```diff
@@ -1046,9 +1046,13 @@ String DatabaseCatalog::getPathForDroppedMetadata(const StorageID & table_id) co
 
 String DatabaseCatalog::getPathForMetadata(const StorageID & table_id) const
```

Comment:
Should we change `getPathForDroppedMetadata` too?

Reply:
Here we have `metadata/*database_name*` symlink, which we use to move the dropped metadata file back to where it originally was. On the other hand in `getPathForDroppedMetadata` we have just a folder `metadata_dropped/` with SQL files with metadata in it, so I am not even sure what we should change it to.